tests/ui/mir/mir_let_chains_drop_order.stderr

```diff
@@ -0,0 +1,40 @@
+warning: unexpected `cfg` condition name: `edition2024`
```

Comment:
Problem [REV6]: that's strange, I would've expected compiletest to register the cfg automatically, I'll take a closer took tomorrow.

Reply:
I am also looking at it.

Reply:
Oh it is a false alarm. It might get included a long while back this `stderr` file should not have been a part of the test suite.